src/services/document-service.js

```diff
@@ -324,6 +330,23 @@ class DocumentService {
     }
   }
 
+  async toggleDocumentArchived({ documentKey, user, archived }) {
```

Comment:
This name is misleading, a toggle would work in a sense that it gets the last state and toggles it to the other state.
Mayne setArchivedState?

Reply:
Yes makes sense, renamed.